lib/doekbase/data_api/tests/examples/get_core_exons_for_gene.py

```diff
@@ -0,0 +1,92 @@
+
```

Comment:
get_core_exons_for_gene.py needs comments that explain what is being done and why